admission-webhook/.travis.sh

```diff
@@ -23,6 +23,10 @@ main() {
 }
 
 run_integration_tests() {
+    if [ "$WITHOUT_ENVSUBST" ] && [ -x "$(command -v envsubst)" ]; then
+        sudo rm -f "$(command -v envsubst)"
```

Comment:
Last nit (promised!): it's nice to be able to run tests on a system and expecting them not to do unexpected changes to it. Since we already have a WITHOUT_ENVSUBST env var, might as well use it in write_manifests_file below to use docker regardless of envsusbt being around instead of deleting envsubst?

Or if you want to be extra thorough, still delete it if `[[ "$TRAVIS" == 'true' ]]` (so that Travis does really test  envsusbt not being around), _and_ don't use it write_manifests_file below if WITHOUT_ENVSUBST is defined, so that one can still run tests on another system and not modify it in expected ways?

Reply:
I'm sorry, I want to make sure I understand what you meant. So add another check before deleting envsubst to only delete it if it's being run under travis, and then add a check to write_manifests_file and run docker regardless of whether envsubst exists or not, if WITHOUT_ENVSUBST is set?

Reply:
Yes :)

Allows Travis to test envsubst not being around, and allows testing elsewhere without actually deleting envsubst.

Sorry for not being clear upfront.